server/tests/routes/user.test.js

```diff
@@ -0,0 +1,146 @@
+import request from 'supertest-as-promised';
+import httpStatus from 'http-status';
+import chai, { expect } from 'chai';
+import app from '../../../index';
+import userService from '../../services/user';
+
+chai.config.includeStack = true;
+
+describe('## User APIs', () => {
+  let user = {
+    name: 'Jarryd Lee',
+    email: 'abc@google.com'
+  };
+
+  const invalidUser = {
+    name: 'Jarryd Lee',
+    email: 0
+  };
+
+  describe('# POST /api/v1/user', () => {
```

Comment:
The reason the `let user = ...` is above was so that the next test has a user to GET, but I think it would be better to just put the creation in a `before(() => {})` block and move the users' in as you suggested